chrome.tera

```diff
@@ -0,0 +1,48 @@
+---
+whiskers:
+  version: 2.0.0
+  matrix:
+    - flavor
+    - accent
+  filename: "themes/{{ flavor.identifier }}/{{ accent }}/manifest.json"
+version: 4.1
+---
+{%- set palette = flavor.colors -%}
+{%- set identCap = flavor.identifier | capitalize -%}
+{%- set accentCap = accent | capitalize -%}
+{%- macro rgb(c) -%}
+[{{c.rgb.r}}, {{c.rgb.g}}, {{c.rgb.b}}]
+{%- endmacro rgb -%}
+{
+  "manifest_version": 3,
+  "name": "Catppuccin Chrome Theme - {{ identCap }} {{ accentCap }}",
+  "description": "Soothing pastel theme for Google Chrome - Catppuccin {{ identCap }}",
```

Comment:
```suggestion
{%- macro rgb(c) -%}
[{{c.rgb.r}}, {{c.rgb.g}}, {{c.rgb.b}}]
{%- endmacro rgb -%}
{
  "manifest_version": 3,
  "name": "Catppuccin Chrome Theme - {{ flavor.name }} {{ accent | capitalize }}",
  "description": "Soothing pastel theme for Google Chrome - Catppuccin {{ flavor.name }}",
```

Reply:
Diff doesn't look great but we can use `flavor.name` instead of `identCap`, and since we are only capiatlizing once no need for a variable.

Reply:
I think this was a remnant from when I was trying to make both variants (now 2 templates) in one :+1: